.github/scripts/install-arduino-core-esp32.sh

```diff
@@ -15,13 +15,8 @@ if [ ! -d "$ARDUINO_ESP32_PATH" ]; then
 		pip install requests > /dev/null
 	fi
 
-	if [ "$GITHUB_REPOSITORY" == "espressif/arduino-esp32" ];  then
-		echo "Linking Core..."
-		ln -s $GITHUB_WORKSPACE esp32
-	else
-		echo "Cloning Core Repository..."
-		git clone https://github.com/espressif/arduino-esp32.git esp32 > /dev/null 2>&1
-	fi
+	echo "Linking Core..."
+	ln -s $GITHUB_WORKSPACE "$ARDUINO_ESP32_PATH"
```

Comment:
please do not edit those scripts! you do not need them! they are for CI and specific tasks.

Reply:
I understand what they are for, please take a look once at my edits and imagine the actual functioning of these scripts from a forked repo.

When the CI pipeline is ran from github.com/timkoers/arduino-esp32.git, the repository owner expects it's repository to be used during the CI pipeline. As of your original .sh files, the github.com/espressif/arduino-esp32.git repository is cloned and the various tests will be ran on the espressif/arduino-esp32 repository, **NOT** on the owners/forkers repository.

I had been searching for over 60+ minutes on why the various tests did result in anything on my repo, so please kindly accept this since it will not break anything on your side

Reply:
tests can run outside of github CI. in that case `GITHUB_REPOSITORY` will not be defined. Maybe detect if in Github CI instead of checking if `GITHUB_REPOSITORY` is the original repository. I can accept that.

Reply:
how about you revert all changes to `.github/` files and open a new PR for that? Already having the PR here means that your changes do go through CI anyway

Reply:
So a simple,

	if [ ! -z "$GITHUB_REPOSITORY" ];  then
		echo "Linking Core..."
		ln -s $GITHUB_WORKSPACE esp32

Would suffice?